sdk/va-report-components/documentation/docs/api/setFormatterLocale.md

```diff
@@ -0,0 +1,30 @@
+---
+id: setFormatterLocale
+title: setFormatterLocale
+---
+
+```
+setFormatterLocale(locale): void
+```
+
+`setFormatterLocale` enables the user to set a language to be used for formatting numeric values and dates. If not specified this will fallback to either the language specified by calling setLocale, or the language specified in the browser settings.
```

Comment:
Just a couple of tweaks to help L10N... enables the user to set the language that is used to format numeric values and dates. If not specified, the language will fallback to the one that is specified by calling setLocale or the one is specified in the browser settings.

Reply:
Done. Thanks.